tests/returntypes/test_parameter_shift_shot_vec_new.py

```diff
@@ -1288,14 +1315,15 @@ def test_involutory_variance_single_param(self, tol):
 
         assert len(tapes) == 1 + 2 * 1
 
-        tapes, fn = qml.gradients.finite_diff(tape)
-        gradF = fn(dev.batch_execute(tapes))
+        tapes, fn = qml.gradients.finite_diff(tape, h=10e-2, shots=shot_vec)
+        all_gradF = fn(dev.batch_execute(tapes))
         assert len(tapes) == 2
 
         expected = 2 * np.sin(a) * np.cos(a)
 
-        # TODO: finite diff shot-vector update
-        # assert gradF == pytest.approx(expected, abs=shot_vec_tol)
+        for gradF in all_gradF:
+            assert gradF == pytest.approx(expected, abs=finite_diff_tol)
```

Comment:
Any reason for using `pytest.approx` over `np.allclose`? I know this was already being used in this file, but just curious 🤔

Reply:
No, remnant of how the previous tests were devised.